README.md

```diff
@@ -170,7 +170,7 @@ In TypeScript (requires LangChain.js version 0.0.166 or later):
 import { RemoteRunnable } from "langchain/runnables/remote";
 
 const chain = new RemoteRunnable({
-  url: `http://localhost:8000/chain/invoke/`,
```

Comment:
Actual bug here

Reply:
Results in a call to `/chain/invoke/invoke`

Reply:
good catch! thank you!